protos/sources/vega/governance.proto

```diff
@@ -59,6 +59,10 @@ message NewMarketConfiguration {
   // Percentage move up and down from the mid price which specifies the range of 
   // price levels over which automated liquidity provision orders will be deployed
   string lp_price_range = 8;
+  // Linear slippage factor is used to cap the slippage component of maintainence margin - it is applied to the slippage volume
```

Comment:
**thought (non-blocking)**: Why do the protos get better comments than the GraphQL fields? 😢

Reply:
I'll copy them over on the followup PR. thanks.